lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -3061,7 +3061,8 @@ void DWARFASTParserClang::ParseSingleMember(
   // artificial member with (unnamed bitfield) padding.
   // FIXME: This check should verify that this is indeed an artificial member
   // we are supposed to ignore.
-  if (attrs.is_artificial) {
+  if (attrs.is_artificial &&
+      !TypeSystemClang::IsCoroutineFrameType(class_clang_type)) {
```

Comment:
I do still wonder if we should just make a `ShouldIgnoreArtificialField` API that returns `true` for known fields we want to skip. The only ones that comes to mind are artificial variables that start with `_vptr$`

Not sure if there are some Objective-C artificial vars we want to skip. Maybe @jimingham or @adrian-prantl would know

Reply:
I changed to `ShouldIgnoreArtificialField` per suggest (unite tests are passed). It looks like the vtable pointer is the only case for C/C++.